From: Michal Schmidt <mschmidt@redhat.com>
Date: Thu, 2 Sep 2010 23:23:39 -0400
Subject: [net] ifb: fix syntax error in pr_debug usage
Message-id: <20100902232339.10621.80626.stgit@brian.englab.brq.redhat.com>
Patchwork-id: 28036
O-Subject: [RHEL5.6 BZ629634 PATCH 1/5] ifb: fix syntax error in pr_debug usage
Bugzilla: 629634
RH-Acked-by: David S. Miller <davem@redhat.com>
RH-Acked-by: Jiri Pirko <jpirko@redhat.com>
RH-Acked-by: Stanislaw Gruszka <sgruszka@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

The missing comma would cause trouble when pr_debug will be changed to
check for compile errors even if !DEBUG.

from upstream 8057de64fd4734ae3e70cf76deb77f1c19958494

diff --git a/drivers/net/ifb.c b/drivers/net/ifb.c
index 43e3f33..24bf89d 100644
--- a/drivers/net/ifb.c
+++ b/drivers/net/ifb.c
@@ -200,7 +200,7 @@ static struct net_device_stats *ifb_get_stats(struct net_device *dev)
 
 	pr_debug("tasklets stats %ld:%ld:%ld:%ld:%ld:%ld:%ld:%ld:%ld \n",
 		dp->st_task_enter, dp->st_txq_refl_try, dp->st_rxq_enter, 
-		dp->st_rx2tx_tran dp->st_rxq_notenter, dp->st_rx_frm_egr,
+		dp->st_rx2tx_tran, dp->st_rxq_notenter, dp->st_rx_frm_egr,
 		dp->st_rx_frm_ing, dp->st_rxq_check, dp->st_rxq_rsch );
 
 	return stats;
